Code/LauncherUnified/Platform/Windows/Launcher_Traits_Windows.h

```diff
@@ -10,3 +10,4 @@
 #define AZ_TRAIT_LAUNCHER_LOWER_CASE_PATHS 0
 #define AZ_TRAIT_LAUNCHER_USE_CRY_DYNAMIC_MODULE_HANDLE 0
 #define AZ_TRAIT_SHARED_LIBRARY_FILENAME_FORMAT R"(%s\%s.dll)"
+#define AZ_TRAIT_CONSOLE_MODE_SUPPORT 0
```

Comment:
Nothing in this code seems to suggest why this cant work on Windows too. Why not have ConsoleMode for all steam platforms (Linux, pc, Mac).

Reply:
Its explained for windows at least why its not supported in the [RFC](https://github.com/o3de/sig-simulation/issues/89) uner 'What are the disadvantages of the feature?'.  When applications are built for windows, it will be built as a console executable or application (the difference is whether or not it uses `main` or `WinMain`. So when you launch it from command line or double click, it launches a new process and the command returns immediately, unlike a console application. 
The only way to support this on Windows is to make this a compile option like how headless servers were done. As for Mac, I didnt want to enable a platform that without proper testing of it, and the use case for this was from simulations on Linux machines.